docs/Node Operators/best-security-practices.md

```diff
@@ -66,13 +72,15 @@ On performing system maintenance to update the Chainlink node, follow [this](/do
 The following are suggestions for job specifications and configuration settings for the node.
 
 [Job Specifications](../jobs/):
+
 - Include the address of your oracle contract address for all RunLog initiated jobs, as shown in the [Fulfilling Requests](../fulfilling-requests/#add-a-job-to-the-node) guide.
 - Override the global `MIN_INCOMING_CONFIRMATIONS` config by setting a `confirmations` field in jobs which perform off-chain payments to allow for greater security by making the node ensure the transaction is still valid after X blocks.
 
 [Configuring Chainlink Nodes](../configuration-variables/):
+
 - [MINIMUM_CONTRACT_PAYMENT_LINK_JUELS](../configuration-variables/#minimum_contract_payment_link_juels): ensure your required payment amount is high enough to meet the costs of responding on-chain.
 - [MIN_INCOMING_CONFIRMATIONS](../configuration-variables/#min_incoming_confirmations): this can be set to 0 for common data request jobs. See the bullet above on setting individual `confirmations` for specific jobs.
-- [LOG_TO_DISK](../configuration-variables/#log_to_disk): Set to `false` if you're using external log drivers which parse the output from Docker containers. This will save you disk space.
+- [LOG_FILE_MAX_SIZE](../configuration-variables/#log_file_max_size): Set this to `0` if you're using external log drivers which parse the output from Docker containers. This will save you disk space.
```

Comment:
@vyzaldysanchez this isn't correct, is it? From the [CHANGELOG](https://github.com/smartcontractkit/chainlink/blob/release/1.3.0-rc3/docs/CHANGELOG.md):

> LOG_FILE_MAX_SIZE (default: 5120mb) - this env var allows you to override the log file's max size (in megabytes) before file rotation.

Reply:
Setting the value to 0 will definitely prevent debug logs to be stored in disk, since this will override the file size, and having the size as 0 states that no file will be created. @chainchad